src/mscorlib/shared/System/Diagnostics/Tracing/EventSourceException.cs

```diff
@@ -18,9 +18,6 @@ namespace System.Diagnostics.Tracing
     /// <summary>
     /// Exception that is thrown when an error occurs during EventSource operation.
     /// </summary>
-#if !ES_BUILD_PCL
-    [Serializable]
```

Comment:
A couple of things to be aware of:

1. This is shared source so it will sync to CoreRT and ProjectN as well.  You'll want to make sure that you have the proper !CORERT ifdef as well if you care about CoreRT.
2. EventSource tests are located in CoreFx.  Before you submit, you should request that CoreFX tests be run against this change.  These are the trigger phrases you want to use (which will automatically request the runs):

@dotnet-bot test Ubuntu x64 corefx_baseline
@dotnet-bot test Windows_NT x64 corefx_baseline